graphql/parse.lua

```diff
@@ -293,11 +317,11 @@ local graphQL = P {
 
 -- TODO doesn't handle quotes that immediately follow escaped backslashes.
 local function stripComments(str)
-  return (str .. '\n'):gsub('(.-\n)', function(line)
+  return (str .. '\n'):gsub('(.-\n)', function(_line)
     local index = 1
-    while line:find('#', index) do
-      local pos = line:find('#', index) - 1
-      local chunk = line:sub(1, pos)
+    while _line:find('#', index) do
+      local pos = _line:find('#', index) - 1
+      local chunk = _line:sub(1, pos)
```

Comment:
Don't get this change.

Reply:
There is a global [`line`](https://github.com/tarantool/graphql/blob/master/graphql/parse.lua#L5) variable, which was shadowed here.

Reply:
I would rename it to `lineno` or so and don't use one term 'line' for two different things within at least a file.